web-components/src/components/card-v2/CardV2.stories.ts

```diff
@@ -26,6 +27,20 @@ const render = (args: Args) => {
         action("expand-card-toggled")(e.detail);
       }}
     >
+      <div slot="card-extra-info">
+        <md-icon slot="icon" name="arrow-tail-up-bold" color="green-50" size="18" iconSet="momentumDesign"></md-icon>
+        <span>This is a test</span>
```

Comment:
do not use green-50 these are legacy colours you can use the variable --md-alert-success-text-color